api/config.py

```diff
@@ -140,12 +144,32 @@ class Configuration(CoreConfiguration):
     LIBRARY_SETTINGS = CoreConfiguration.LIBRARY_SETTINGS + [
         {
             "key": LIBRARY_DESCRIPTION,
-            "label": _("A short description of this library, shown to people who aren't sure they've chosen the right library."),
+            "label": _("A short description of this library."),
+            "description": _("This will be shown to people who aren't sure they've chosen the right library."),
+            "optional": True,
+        },
+        {
+            "key": HELP_EMAIL,
+            "label": _("Patron support email address"),
+            "description": _("An email address a patron can use if they need help, e.g. 'simplyehelp@yourlibrary.org'."),
+            "optional": True,
+        },
+        {
+            "key": HELP_WEB,
+            "label": _("Patron support web site"),
+            "description": _("A URL for patrons to get help."),
+            "optional": True,
+        },
+        {
+            "key": HELP_URI,
+            "label": _("Patron support custom integration URI"),
+            "description": _("A custom help integration like Helpstack, e.g. 'helpstack:nypl.desk.com'."),
```

Comment:
I moved these fields above the "designated agent" field because otherwise the description of that field, which mentions the generic patron support address, is unclear.